test/black-box/address/AddressListCommand.test.ts

```
@@ -70,7 +70,9 @@ describe('AddressListCommand', () => {
     test('Should return code 500 when no profile is given', async () => {
```

Should fail to list addresses when missing profileId

---

test description does not match the functionality